### DISCUSSION TITLE
Hourly data API

### DISCUSSION THREAD

**Discussion by utsavvasoya** [amzn/ads-advanced-tools-docs#184] [General]
Does Amazon provide hourly data API like a reporting API (sales/spend/orders/clicks etc..) I am finding an API that will provide me with data every 1 hr. Right now I am using **/reporting/reports**  this API to get data Now I want every 1 hr updated data.

**Comment by elizafarley** [comment]
Hi @utsavvasoya the Amazon Ads reporting API only provides data at the daily and summary levels. If you want hourly data, I would recommend looking into [Amazon Marketing Stream](https://advertising.amazon.com/API/docs/en-us/guides/amazon-marketing-stream/overview) or [Amazon Marketing Cloud](https://advertising.amazon.com/API/docs/en-us/guides/amazon-marketing-cloud/overview#tag/Ad-Group-Targeting-Devices/operation/DspGetAdGroupDeviceTargets). [This post](https://github.com/amzn/ads-advanced-tools-docs/discussions/178#discussioncomment-7492350) provides a helpful overview and comparison between these three reporting solutions.

For hourly retail data, you can also look into [Rapid Retail Analytics](https://developer-docs.amazon.com/sp-api-blog/docs/optimize-your-business-using-rapid-retail-analytics-data#metric-definitions) using the Selling-Partner API.

**Comment by utsavvasoya** [comment]
Thanks, I'll check it out.
